src/utils/number_utils.ts

```diff
@@ -6,12 +6,29 @@ FLOAT[0] = -1;
 // Big endian    [191, 240, 0, 0, 0, 0, 0, 0]
 const isBigEndian = FLOAT_BYTES[7] === 0;
 
+/**
+ * @experimental
+ * @public
+ */
+export type NumberUtils = {
```

Comment:
Can we add API docs to the `NumberUtils` type, to clarify what it encapsulates? Just a sentence or two.